haystack/components/audio/whisper_remote.py

```diff
@@ -15,12 +15,20 @@
 @component
 class RemoteWhisperTranscriber:
     """
-    Transcribes audio files using OpenAI's Whisper using OpenAI API. Requires an API key. See the
-    [OpenAI blog post](https://beta.openai.com/docs/api-reference/whisper) for more details.
-    You can get one by signing up for an [OpenAI account](https://beta.openai.com/).
+    Transcribes audio files using the Whisper API from OpenAI.
 
+    The component requires an API key, see the relative
+    [OpenAI blog post](https://beta.openai.com/docs/api-reference/whisper) for more details.
```

Comment:
the URL is outdated but redirects to the API reference.
I would propose linking this: https://platform.openai.com/docs/api-reference/authentication